packages/core/src/domain/report/reportObservable.ts

```diff
@@ -0,0 +1,119 @@
+import { mergeObservables, Observable } from '../../tools/observable'
+import { DOM_EVENT, includes, addEventListener } from '../../tools/utils'
+import { monitor } from '../internalMonitoring'
+import type {
+  Report,
+  BrowserWindow,
+  ReportType,
+  ReportingObserverCallback,
+  ReportingObserverOption,
+  ReportingObserver as ReportingObserverInterface,
+} from './browser.types'
+
+declare const ReportingObserver: {
```

Comment:
I tried to move the ReportingObserver type declaration in the `browser.types` without success. I run into several typescript issues. This is the best solution I could found.